Mage.Sets/src/mage/cards/c/CompoundFracture.java

```diff
@@ -0,0 +1,46 @@
+package mage.cards.c;
+
+import mage.abilities.dynamicvalue.DynamicValue;
+import mage.abilities.dynamicvalue.common.CardsInControllerGraveyardCount;
+import mage.abilities.effects.common.continuous.BoostTargetEffect;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.Duration;
+import mage.filter.FilterCard;
+import mage.filter.predicate.mageobject.NamePredicate;
+import mage.target.common.TargetCreaturePermanent;
+
+import java.util.UUID;
+
+/**
+ * @author tiera3 - modified from Growth Cycle
+ */
+public final class CompoundFracture extends CardImpl {
+
+    private static final FilterCard filter = new FilterCard("card named Compound Fracture");
+    static {
+        filter.add(new NamePredicate("Compound Fracture"));
+    }
+    private static final DynamicValue xValue = new CardsInControllerGraveyardCount(filter, -1);
```

Comment:
For info, not pr. If you see xValue then it’s a good point to add some card hint about calculated value for a user. Current example: dynamic card hint with “com fra cards in your graveyard: 123”